hack/get-cd-component-name.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+#
+# Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+echo "github.com/gardener/gardener-extension-networking-calico"
```

Comment:
Is it really necessary to have a bash script just for one configuration setting? Would it not be better to have multiple config options in one file together and source it?

Reply:
As proposed, now all configuration settings are in one file which is sourced.